bots/common/src/main/java/org/openjdk/skara/bots/common/CommandNameEnum.java

```diff
@@ -25,6 +47,19 @@ public enum CommandNameEnum {
     backport,
     tag;
 
+    private boolean isMultiLine = false;
+
+    CommandNameEnum() {
+    }
+
+    CommandNameEnum(boolean isMultiLine) {
+        this.isMultiLine = isMultiLine;
+    }
+
+    public boolean isMultiLine() {
+        return isMultiLine;
+    }
+
```

Comment:
Introduced a field called` isMultiLine` here because `CommandHandler` is coupled in PR bot and it is very difficult to extract it to the bots::common package.